.cloud-build/execute_changed_notebooks_helper.py

```diff
@@ -156,9 +157,10 @@ def _create_tag(filepath: str) -> str:
     return tag
 
 
-rate_limit = RateLimit(max_count=10, per=60, greedy=True)
-
+rate_limit = RequestRate(10, Duration.HOUR)
+limiter = Limiter(rate_limit)
```

Comment:
Make sure you use limiter = Limiter(bucket_class=FileLockSQLiteBucket)
As that one is thread-safe.

Reply:
done